tests/InputFiles/Elasticity/SingleCoatingMirror.yaml

```diff
@@ -0,0 +1,174 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+Executable: SolveElasticity3D
```

Comment:
Does this mean we build yet another executable in CI? If so, can we replace an existing build instead?

Reply:
No the HalfSpaceMirror.yaml already builds this exec.